tests/integration/test_types_integration.py

```diff
@@ -737,13 +739,30 @@ def create_timezone(timezone_str: str) -> tzinfo:
         return ZoneInfo(timezone_str)
 
 
-def test_interval(trino_connection):
+def test_interval_year_to_month(trino_connection):
     SqlTest(trino_connection) \
         .add_field(sql="CAST(null AS INTERVAL YEAR TO MONTH)", python=None) \
+        .add_field(sql="INTERVAL '10' YEAR", python=relativedelta(years=10)) \
```

Comment:
Please test for min/max valid trino value and min/max valid python values too.

Reply:
Does this test verify both directions? i.e. converting from Trino string representation to Python object AND converting from Python object to Trino SQL (e.g. queries with parameters?)?